tm2/pkg/bft/rpc/core/abci.go

```diff
@@ -56,6 +60,15 @@ import (
 // | height    | int64  | 0       | false    | Height (0 means latest)                        |
 // | prove     | bool   | false   | false    | Includes proof if true                         |
 func ABCIQuery(ctx *rpctypes.Context, path string, data []byte, height int64, prove bool) (*ctypes.ResultABCIQuery, error) {
```

Comment:
Can you please add a small unit test that verifies the errors?

I know our RPC testing suite is super janky, but it shouldn't be too much of a hassle to write a few cases

Reply:
I promise refactoring this testing suite is on my bucket list